setup.py

```diff
@@ -74,7 +75,10 @@ def read(fname):
     tests_require=['mock>=1.0.1', 'httpretty>=0.8.3', ],
 )
 
-pip_main(['install', 'ansible'])
+subprocess.check_call([sys.executable, '-m', 'pip', 'install', '--upgrade pip==9.0.3'])
```

Comment:
does that work?  seems like it would it be `'-m', 'pip', 'install', '--upgrade', 'pip==9.0.3', 'path/to/requirements.txt'`?